integration-tests/Cargo.toml

```diff
@@ -19,6 +19,7 @@ bytesize.workspace = true
 borsh.workspace = true
 chrono.workspace = true
 clap.workspace = true
+derive_builder.workspace = true
```

Comment:
Does it work if added under `dev-dependencies`? As it is for tests only.

Reply:
Yep, correct